packages/react/src/components/Card/__stories__/Card.stories.js

```diff
@@ -32,19 +34,22 @@ export default {
         ),
         copy: text('copy', '', groupId),
         inverse: boolean('inverse', false, groupId),
-        cta: {
-          href: text('Cta href (cta.href)', 'https://example.com', groupId),
-          copy: text('Cta copy (cta.copy)', 'Card cta text', groupId),
-          icon: {
-            src: ArrowRight20,
-          },
-          iconPlacement: select(
-            'Cta icon placement (cta.iconPlacement)',
-            ['left', 'right'],
-            'right',
-            groupId
-          ),
-        },
+        cta: !DDS_CARD_WITH_PICTOGRAM
```

Comment:
@guilhermelMoraes one thought, not sure if we should be doing this logic based on the setting of the flag. Instead, we should probably check if the pictogram is defined or not. An adopter might want to have a mix of cards with pictograms and cards with CTAs on the same application.